test/integration/prefs.js

```diff
@@ -287,7 +287,7 @@ async function testChangeMSPrompt() {
   process.argv = ['/usr/local/nodejs/bin/node',
     '/usr/local/nodejs/bin/autolabjs', 'prefs', 'changeserver', '--type', 'ms'];
   const mockInquirer = sandbox.mock(inquirer);
-  const testPort = 5687;
+  const testPort = '5687';
```

Comment:
Is there any specific reason for this change?

Reply:
This description here explains the change https://github.com/chriso/validator.js#strings-only 

> This library validates and sanitizes strings only.
> If you're not sure if your input is a string, coerce it using input + ''. Passing anything other than a string is an error.

Honestly, this is because of the #108. I think I have an updated version of `validator.js` in my `node_modules`.